include/Route.hpp

```diff
@@ -13,14 +13,23 @@ using namespace std;
 using namespace httplib;
 using namespace nlohmann;
 
+/// @brief Key/Value pair that maps authorized user ids to their token.
 map<int, string> authedUsers;
 
+/// @brief Super class for managing all the endpoints of the API.
 class Route {
  protected:
+    /// @brief Shared pointer to the database.
     shared_ptr<Database> db;
+
+    /// @brief Shared pointer to the webserver object.
     shared_ptr<Server> server;
+
+    /// @brief Shared pointer to the prediction manager.
     PredictionManager manager;
```

Comment:
these are not particularly needed, its like ah yes, this getter gets the object

I think you can omit these comments